gnome-initial-setup/pages/live-chooser/eos-setup-live-user

```diff
@@ -178,6 +165,18 @@ class SetupLiveUser(object):
         self.update(GS_SCHEMA, ALLOW_UPDATES, GLib.Variant('b', False))
 
 
+def install_desktop_file():
```

Comment:
(By the way, thanks for making this a non-member function of the class - it was one of the things that was initially annoying me about this code when I first worked with it ;-) )